tests/plugins/test_koji_import.py

```diff
@@ -926,7 +926,7 @@ def check_components(components):
         assert len(components) > 0
         for component_rpm in components:
             assert isinstance(component_rpm, dict)
-            assert set(component_rpm.keys()) == set([
+            assert set(component_rpm.keys()) == {
```

Comment:
Why didn't this (and the subsequent calls to  `set` in the other assertions) become a literal as well?